modules/aws-eks-teams/main.tf

```diff
@@ -198,15 +206,14 @@ resource "aws_iam_role" "platform_team" {
   tags = var.tags
 }
 
-# ---------------------------------------------------------------------------------------------------------------------
-# Platform Team EKS access IAM policy
-# ---------------------------------------------------------------------------------------------------------------------
+# Platform Team EKS Access IAM policy
+resource "aws_iam_policy" "platform_team_iam_policy" {
+  count = length(var.platform_teams) > 0 ? 1 : 0
 
-resource "aws_iam_policy" "platform_team_eks_access" {
-  count       = length(var.platform_teams) > 0 ? 1 : 0
-  name        = "${var.eks_cluster_id}-PlatformTeamEKSAccess"
+  name        = "${var.eks_cluster_id}-platform-team-iam-policy"
```

Comment:
I think the previous name was good and followed AWS policy naming convention, it was clear defining the purpose of the policy.
What is the reason for the suggested change?